parser/testdata/fuzz/FuzzParseSipMessage/2c025c3ae292cb8b

```diff
@@ -0,0 +1,2 @@
+go test fuzz v1
```

Comment:
I guess this was failing examples, but now they are fixed. 
Do we really need to hold on this?
Files are just very autogenerated and I guess they can be created with fuzz test every time (if not fixed already)

Reply:
These files become test cases that help verify that the same kind of crash is not reintroduced into the parser later, so it's a free test case as such. If you undo one of my changes and run `go test` you'll see that the test fun fails.

These files are based on crashing cases that the fuzzer has found, they will not be generated just from new runs of the parser after the crash has been fixed.

I agree that the fuzzer ends up generating very quirky examples. Part of the reason is that it works hard to make the test cases as minimal as possible.